netebpfext/net_ebpf_ext.h

```diff
@@ -36,8 +36,12 @@
 #define NET_EBPF_EXTENSION_POOL_TAG 'Nfbe'
 #define NET_EBPF_EXTENSION_NPI_PROVIDER_VERSION 0
 
+CONST IN6_ADDR DECLSPEC_SELECTANY in6addr_v4mappedprefix = IN6ADDR_V4MAPPEDPREFIX_INIT;
+
 #define htonl(x) _byteswap_ulong(x)
 #define htons(x) _byteswap_ushort(x)
+#define ntohs(x) htons(x)
+#define ntohl(x) htonl(x)
 
```

Comment:
```suggestion
#define ntohl(x) _byteswap_ulong(x)
#define ntohs(x) _byteswap_ushort(x)
```
nit: alphabetize, and define the same as htonl etc but semantically different so don't define one as the other